readme.md

```diff
@@ -0,0 +1,68 @@
+<snippet>
+<content>
+
+[![Build Status](https://travis-ci.org/andela-hoyeboade/amity_room_allocator.svg?branch=working)](https://travis-ci.org/andela-hoyeboade/amity_room_allocator) [![Coverage Status](https://coveralls.io/repos/github/andela-hoyeboade/amity_room_allocator/badge.svg?branch=working)](https://coveralls.io/github/andela-hoyeboade/amity_room_allocator?branch=working)
+
+# Amity Room Allocator
+Python console application for allocating fellows and staffs to offices or living spaces in Amity.
```

Comment:
**README:** Give more context about the project e.g. checkpoint 1, link to the checkpoint doc, etc.